Makefile

```diff
@@ -15,44 +16,40 @@ goto_src=cd ${src_dir} && source ${venv_dir}/bin/activate
 help:
 	@echo "Make tasks for installation and deployment. Refer to the Makefile content to know more."
 
-logs:
-	@${remote} "tail -f ~/log/preprod.log"
+logs:  # type=error|access
+	@${remote} "tail -f ~/log/${type}.log"
 
-flush-logs:
-	@${remote} "echo '' > ~/log/preprod.log"
+flush-logs:  # type=error|access
+	@${remote} "echo '' > ~/log/${type}.log"
 	@echo "Log file emptied!"
 
-deploy:
-	@echo "> Fetching master branch and updating sources."
+deploy:  # env=prod|preprod
+	@echo "\n> Fetching master branch and updating sources."
 	${remote} "${goto_src} && git fetch origin ${branch} && git checkout ${branch} && git reset --hard FETCH_HEAD"
 	${remote} "${goto_src} && pip install -r requirements.txt"
-	make rsync
-	@echo "> Launching gunicorn daemon."
-	${remote} "${goto_src} && pkill gunicorn; \
-		gunicorn --daemon -b 127.0.0.1:5000 preprod:app \
-		--error-logfile ~/log/preprod.log --access-logfile ~/log/preprod.log"
-	@echo "> App is deployed. Run \`make logs\` to follow activity."
-
-rsync:
-	@echo "> Synchronizing archives media from old production server."
+	make sync_archives_media
+	@echo "\n> Launching gunicorn daemon."
+	${remote} "${goto_src} && pkill gunicorn 2> /dev/null; \
+		gunicorn --daemon -b ${${env}_server} ${env}:app \
+		--error-logfile ~/log/error.log --access-logfile ~/log/access.log"
+	@echo "\n> App is deployed. Run \`make logs env=${env} type=access|error\` to follow activity."
+
+sync_archives_media:
+	@echo "\n> Synchronizing archives media from old production server."
 	${remote} "rsync --archive --compress --human-readable --inplace --progress ${old_prod}:${old_prod_media_dir}/avatars/ ${src_dir}/cafebabel/uploads/users/"
 	${remote} "rsync --archive --compress --human-readable --inplace --progress ${old_prod}:${old_prod_media_dir}/editorials/ ${src_dir}/cafebabel/uploads/articles/"
 	${remote} "rsync --archive --compress --human-readable --inplace --progress ${old_prod}:${old_prod_media_dir}/cache/ ${src_dir}/cafebabel/uploads/archives/"
 
-install:
-	@echo "> Installing sources, dependencies and database."
+install:  # env=prod|preprod
+	@echo "\n> Installing sources, dependencies and database."
 	-${remote} "git clone https://github.com/cafebabel/cafebabel.com.git ${src_dir}"
 	-${remote} "python3.6 -m venv ${venv_dir}"
 	-${remote} "${goto_src} && make make-dirs"
-	make reset-db
 	make deploy
 
 make-dirs:
 	mkdir -p ./logs
+	mkdir -p ./cafebabel/uploads/{archives,articles,tags,users}
 
-reset-db:
```

Comment:
If we remove this, we have to document Flask commands to launch for setting up the database?

Reply:
Right, let's keep it

Reply:
Can you do it?

Reply:
Oups, I had in mind I had pushed that 😅